src/sentry/static/sentry/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -202,16 +269,15 @@ class ColumnEditCollection extends React.Component<Props, State> {
     };
     const ghost = (
       <Ghost ref={this.dragGhostRef} style={style}>
-        {this.renderItem(col, index, true)}
+        {this.renderItem(col, index, true, true)}
```

Comment:
Would these args be better as an object so that readers can see what arguments you're passing in?

Reply:
I have another set of changes coming to integrate this with the page, I'll fix this then.